frappe/public/js/frappe/views/reports/query_report.js

```diff
@@ -1371,6 +1371,7 @@ frappe.views.QueryReport = class QueryReport extends frappe.views.BaseList {
 			columns: this.get_columns_for_print(print_settings, custom_format),
 			original_data: this.data,
 			report: this,
+			is_standard_format: this.report_doc.is_standard === "Yes" && custom_format ? 1 : 0,
```

Comment:
this additional check for is_standard is for custom reports created from standard reports

Reply:
it's probably better to offload this to user somehow? :eyes: - in the printview itself?